GCN Files/NK_canonical_linear.gcn

```diff
@@ -0,0 +1,75 @@
+###########################################################################
+# New Keynesian model (Galì, 2008)
+###########################################################################
+
```

Comment:
I added an options flag you can use in linear models that will automatically pass `linear_model = True` to places like `steady_state` and `solve_model`. You can see an example here: https://github.com/jessegrabowski/gEconpy/blob/main/tests/Test%20GCNs/RBC_Linearized.gcn